### PR TITLE
Add warning when transient exception suppression is unavailable due…

### PR DESCRIPTION
**Pull request by alamar**
…to locale

see #172

### CODE REVIEW

File: src/main/java/net/openhft/chronicle/network/TcpEventHandler.java [hunk 1]
```diff
@@ -480,6 +481,8 @@ private void handleIOE(@NotNull final IOException e) {
             } else if (message != null && message.startsWith("An existing connection was forcibly closed")) {
                 Jvm.debug().on(getClass(), message);
             } else if (!(e instanceof ClosedByInterruptException)) {
+                Language.warnOnce();
```
Comment (JerryShea) [inline src/main/java/net/openhft/chronicle/network/TcpEventHandler.java]: I like this very much. Is it worth considering centralising this into Nick's SocketExceptionUtil so we have a one-stop-shop for exception message-related stuff?

**Comment by nicktindall** [comment]
I like the idea, but I don't think it's necessary as we use getMessage instead of getLocalizedMessage. I added [a test](https://github.com/OpenHFT/Chronicle-Network/pull/173/files#diff-89c667bfb6c0b2700b9f32b57c803805485f232f8c9c35a5374845264bc3cdd1R54) that should protect us against different locales and java versions (I've confirmed it passes in Java 8 and 17 with their different behaviours)

**Comment by alamar** [comment]
What do you think about that @nicktindall ?

You need to install `language-pack-xx` and set `LANGUAGE=xx` in order to see the effect in your test

**Comment by nicktindall** [comment]
I'm probably less familiar with languages other than English in the JVM than you are and how common having language packs installed is. 

The primary reason for my change was to stop the test flappiness, but it's probably not the end of the world if you get a warning when the other end drops the connection.

Perhaps the correct fix was to remove the suppression altogether, but that would have meant either adding lots of `ignoreException`s in the tests, or downgrading the log level for the potentially more interesting exceptions.

At the end of the day, identifying exceptions by log messages is a dirty hack and shouldn't be done.

I'll leave it up to you whether to merge this one. I have no objection.

**Review by nicktindall — APPROVED** [review]

**Review by JerryShea — APPROVED** [review]

**Comment by alamar** [comment]
@nicktindall @JerryShea I have pushed an updated commit where the check and warning are moved alongside the Nick's changes. If there are no objections, I will merge tomorrow.

Having language pack is the default for Linux distributions installed in regular fashion by using an installer. Now, in the age of Docker, they should be less common but still prevalent on developer machines. Not sure about the Mac & Windows.

**Comment by nicktindall** [comment]
> @nicktindall @JerryShea I have pushed an updated commit where the check and warning are moved alongside the Nick's changes. If there are no objections, I will merge tomorrow.
> 
> Having language pack is the default for Linux distributions installed in regular fashion by using an installer. Now, in the age of Docker, they should be less common but still prevalent on developer machines. Not sure about the Mac & Windows.

Looks good!

**Pull request merged**